src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -1084,3 +1084,33 @@ extern "C" EXPORT_API void EXPORT_CC coreclr_unity_profiler_register(const CLSID
 
     g_profControlBlock.storedProfilers.InsertHead(profilerData);
 }
+
+extern "C" EXPORT_API gboolean EXPORT_CC coreclr_unity_gc_concurrent_mode(gboolean state)
```

Comment:
The API is such that it returns the previous value?

Reply:
Yes, it returns the previous value. So that you can restore it after an operation that requires disabled concurrent GC can restore the state.